3_javascript/javascript1/week1/script3.js

```diff
@@ -0,0 +1,33 @@
+function calculateHousePrice(width, height, depth, gardenSize) {
+    let volume = width * height * depth;
+    let housePrice = volume * 2.5 * 1000 + gardenSize * 300;
+    return housePrice;
+  }
+  
+  function comparePrices(expectedPrice, actualPrice, personName) {
+    if (expectedPrice > actualPrice) {
+      return personName + " is paying too much for the house.";
+    } else {
+      return personName + " isn't paying too much for the house.";
+    }
+  }
+  
+    let peterWidth = 8;
```

Comment:
you've used let for all the variables. Since the dimensions of Peter's and Julia's houses and the sizes of their gardens are not expected to change, you could consider using const for these variables. This helps to convey the intention that these values should remain constant 😄